src/main/java/org/jabref/gui/groups/GroupNodeViewModel.java

```diff
@@ -59,13 +63,15 @@
     private final CustomLocalDragboard localDragBoard;
     private final ObservableList<BibEntry> entriesList;
     private final DelayTaskThrottler throttler;
+    private boolean displayItemsCount;
 
     public GroupNodeViewModel(BibDatabaseContext databaseContext, StateManager stateManager, TaskExecutor taskExecutor, GroupTreeNode groupNode, CustomLocalDragboard localDragBoard) {
```

Comment:
With constructor call, it's meant as additional parameter:
```suggestion
    public GroupNodeViewModel(BibDatabaseContext databaseContext, StateManager stateManager, TaskExecutor taskExecutor, GroupTreeNode groupNode, CustomLocalDragboard localDragBoard, PreferencesService preferences) {
```

Reply:
@Siedlerchr  Thank you for idea.
I just couldn't find where constructor was used. But it's so simple - just add a parameter and then build the project to see where it breaks. 😂Thanks again.

Reply:
Your IDE should also have this useful function to show the call hierarchy of a method or the references. That also is a useful tool for finding out where it is used

Reply:
thanks.